assets/xml/objects/object_fish.xml

```diff
@@ -1,161 +1,180 @@
-        <DList Name="object_fish_DL_0010A0" Offset="0x10A0" />
-        <DList Name="object_fish_DL_001260" Offset="0x1260" />
-        <DList Name="object_fish_DL_001300" Offset="0x1300" />
-        <DList Name="object_fish_DL_0013A0" Offset="0x13A0" />
-        <Texture Name="object_fish_Tex_0014D0" OutName="tex_0014D0" Format="rgba16" Width="16" Height="16" Offset="0x14D0" />
-        <Texture Name="object_fish_Tex_0016D0" OutName="tex_0016D0" Format="rgba16" Width="16" Height="16" Offset="0x16D0" />
-        <Texture Name="object_fish_Tex_0018D0" OutName="tex_0018D0" Format="rgba16" Width="16" Height="16" Offset="0x18D0" />
-        <Texture Name="object_fish_Tex_001AD0" OutName="tex_001AD0" Format="rgba16" Width="32" Height="32" Offset="0x1AD0" />
-        <Texture Name="object_fish_Tex_0022D0" OutName="tex_0022D0" Format="rgba16" Width="16" Height="16" Offset="0x22D0" />
-        <!-- <Blob Name="object_fish_Blob_0024D0" Size="0x200" Offset="0x24D0" /> -->
-        <Texture Name="object_fish_Tex_0026D0" OutName="tex_0026D0" Format="rgba16" Width="16" Height="16" Offset="0x26D0" />
-        <Limb Name="object_fish_Standardlimb_0028D0" Type="Standard" EnumName="FISHING_FISH_LIMB_01" Offset="0x28D0" />
-        <Limb Name="object_fish_Standardlimb_0028DC" Type="Standard" EnumName="FISHING_FISH_LIMB_02" Offset="0x28DC" />
-        <Limb Name="object_fish_Standardlimb_0028E8" Type="Standard" EnumName="FISHING_FISH_LIMB_03" Offset="0x28E8" />
-        <Limb Name="object_fish_Standardlimb_0028F4" Type="Standard" EnumName="FISHING_FISH_LIMB_04" Offset="0x28F4" />
-        <Limb Name="object_fish_Standardlimb_002900" Type="Standard" EnumName="FISHING_FISH_LIMB_05" Offset="0x2900" />
-        <Limb Name="object_fish_Standardlimb_00290C" Type="Standard" EnumName="FISHING_FISH_LIMB_06" Offset="0x290C" />
-        <Limb Name="object_fish_Standardlimb_002918" Type="Standard" EnumName="FISHING_FISH_LIMB_07" Offset="0x2918" />
-        <Limb Name="object_fish_Standardlimb_002924" Type="Standard" EnumName="FISHING_FISH_LIMB_08" Offset="0x2924" />
-        <Limb Name="object_fish_Standardlimb_002930" Type="Standard" EnumName="FISHING_FISH_LIMB_09" Offset="0x2930" />
-        <Limb Name="object_fish_Standardlimb_00293C" Type="Standard" EnumName="FISHING_FISH_LIMB_0A" Offset="0x293C" />
-        <Limb Name="object_fish_Standardlimb_002948" Type="Standard" EnumName="FISHING_FISH_LIMB_0B" Offset="0x2948" />
-        <Limb Name="object_fish_Standardlimb_002954" Type="Standard" EnumName="FISHING_FISH_LIMB_0C" Offset="0x2954" />
-        <Limb Name="object_fish_Standardlimb_002960" Type="Standard" EnumName="FISHING_FISH_LIMB_0D" Offset="0x2960" />
-        <Limb Name="object_fish_Standardlimb_00296C" Type="Standard" EnumName="FISHING_FISH_LIMB_0E" Offset="0x296C" />
-        <Limb Name="object_fish_Standardlimb_002978" Type="Standard" EnumName="FISHING_FISH_LIMB_0F" Offset="0x2978" />
+        <DList Name="gFishingFishTailFinDL" Offset="0x940"/>
```

Comment:
```suggestion
        <DList Name="gFishingFishTailFinDL" Offset="0x940" />
```
Looks like a lot of places lost a space at the end. I would recommend searching for `"/` to find them.

Reply:
oh wow i didnt even know there was a space there in the other ones lol, the ones without a space are probably the ones i copied over from the oot xml. yea ill search and put them back